caproto/_dbr.py

```diff
@@ -20,11 +20,10 @@
     USE_NUMPY = True
 
 
-class AccessRights(IntEnum):
+class AccessRights(IntFlag):
```

Comment:
nifty! https://docs.python.org/3/library/enum.html#intflag

3.6 is paying divides already.